service/src/main/java/cz/muni/fi/service/DriverService.java

```diff
@@ -1,4 +1,79 @@
 package cz.muni.fi.service;
 
+import cz.muni.fi.entities.Driver;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author Lucie Kureckova, 445264
+ */
+@Service
 public interface DriverService {
-}
+    
+    /**
+     * Find all drivers
+     * 
+     * @return list of drivers
```

Comment:
add `@throws`

Reply:
method doesnt takes any argument so It doesnt throw IllegalArgumentExpresion nor DataAccessException

Reply:
Actually it can throw DataAccessException if Dao layer throw some exception any exception you need to catch it and throw our ServiceDataAccessException which extends it not FormulaTeamDataAccessException